src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -510,6 +510,46 @@ void canDeleteAUser() {
       .statusCode(404);
   }
 
+  @Test
+  void canDeleteAPatronUser() {
+    commitAllMessagesInTopic(TENANT_NAME, USER_CREATED.getTopicName());
+    commitAllMessagesInTopic(TENANT_NAME, USER_DELETED.getTopicName());
+    UserTenant userTenant = new UserTenant()
+      .withId(UUID.randomUUID().toString())
+      .withUserId(UUID.randomUUID().toString())
+      .withUsername("user_test").withTenantId("tenant_test").withCentralTenantId("diku");
+
+    userTenantClient.attemptToSaveUserTenant(userTenant);
+
+    String userId = UUID.randomUUID().toString();
+    final User userToCreate = User.builder()
+      .id(userId)
+      .username("joannek")
+      .active(true)
+      .type("patron")
+      .personal(Personal.builder()
+        .firstName("julia")
+        .preferredFirstName("jules")
+        .lastName("brockhurst")
+        .build())
+      .tags(TagList.builder().tagList(List.of("foo", "bar")).build())
+      .build();
+
+    final var user = usersClient.createUser(userToCreate);
+
+    usersClient.deleteUser(user.getId());
+
+    List<UserEvent> userCreatedEvents = getUserEventsAndFilterByUserId(USER_CREATED, userId);
+    List<UserEvent> userDeletedEvents = getUserEventsAndFilterByUserId(USER_DELETED, userId);
```

Comment:
I think need to add more details maybe in test name and test comments to explain what this test checking.
Like that it checks scenario when we are in consortium mode, and for create events we are not sending events, also need to add some testing for update as well, you can include user update in this test or create new one